web-js/add-timestamp-to-wms-request/README.md

```diff
@@ -0,0 +1,31 @@
+#Add Timestamp to WMS Requests
+
+##About
+Browsers often cache images to boost performance by limiting the number of requests that web applications need to send. While this feature is often helpful, it can be a hinderance in situations where data is constantly being updated. Consider this example. Let's say you are working with a WMS service showing realtime locations of ships. Thus while the extent of the map may not change (triggering no new requests) the data has changed meaning the image needs to be updated. Setting a refresh interval on the WMSLayer is of course the best way to handle this problem. However you will probably find your browser caches the image and does not re-issue the request because the request parameters have not changed. One potential workaround for this issue is to use timestamps.
+
+##Usage Notes
```

Comment:
be explicit about the fact that a proxy is required.  link to the doc to help folks set it up and explain that its the reason we can't host a live demo.